### ISSUE TITLE
Some problems of the store in outposts & loading the game

### ISSUE THREAD

**Issue by CanXue233** [Regalis11/Barotrauma#8365]
- [x] I have searched the issue tracker to check if the issue has already been reported.

Hello, devs.

**Description**
1.When open the store in outposts, the frames of game suddenly dropped, even the game crushed.
2.We noticed that the time to loading the game is much longer than before. Especially during the time when reach or leave the outpost.

**Steps To Reproduce**
Just open the store in the outpost.

**Version**
0.16.6.1   Windows10

**Additional information**
None.

Thanks.

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately wasn't able to get this to happen: the FPS seems to stay at a solid 60 in the store interface. Did this happen in single player or multiplayer? Would it be possible to provide us the crash report?

**Comment by CanXue233** [comment]
In multiplay. But strangely it seems that this is only me, and there is not even a report when the game crashed.
And a new found: the loading speed of Chinese display language is significantly slower than that of English.

**Comment by ekqvist** [comment]
So, this is likely the same issue as #8367?

**Comment by Regalis11** [comment]
Thank you for the clarification!

Fixed in https://github.com/Regalis11/Barotrauma-development/commit/13a508e, we'll try to get this hotfixed soon!

**Comment by Regalis11** [comment]
On a closer look, it seems that the store issue could be unrelated: I couldn't repro it without that fix either, and it seems to be happening to people playing in other languages as well.

**Comment by CanXue233** [comment]
Thanks for the hotfix, devs!

**Comment by Abkim27** [comment]
As I know, framerate drop happens when you are not host?

**Comment by CanXue233** [comment]
Yes, and here are the screenshots. (fps is in the upper left corner)
![20220226185039_1](https://user-images.githubusercontent.com/93201375/155840695-4e49e1e7-ad4f-45ef-a0ef-2a3f407ed75a.jpg)
![20220226185049_1](https://user-images.githubusercontent.com/93201375/155840699-13ea4ba7-d94f-4e78-8849-53066105a31d.jpg)

**Comment by CanXue233** [comment]
Then the game crushed.

**Comment by Rokvach** [comment]
There hasn't yet been any complaints about the store causing performance issues in the Unstable version, but keeping this ticket still open

**Comment by Rokvach** [comment]
No reported cases of this causing any issues, closing.
